doc/project/checklist.md

```diff
@@ -247,46 +247,55 @@ Ensure all these tests verify back-2back access with zero delays, along with par
+Nightly regression for running all tests with multiple random seeds (iterations) setup.
+Selecting the number of iterations for running each test is subjective - it depends on the failure rate and available compute resources.
+For starters, it is recommended to set iterations to 100 for each test.
+It may be trimmed down if the compute load is too high.
+As such, the goal should be to have the nightly regression finish overnight so that the results are available next morning for triage.
 
 ### FPV_REGRESSION_SETUP
 
 Set up FPV regression by adding the module to `hw/formal/fpv_all` script.
 
 ### SIM_COVERAGE_MODEL_ADDED
 
-- Structural coverage collection model checked in. This specifies what hierarchies and what types of coverage to collect. For example, pre-verified sub-mudules (including some `prim` components pre-verified thoroughly with FPV) can be black-boxed and only IO toggle coverage can be setup for those sub-modules for coverage collection.
-- Functional coverage shell object created - this may not contain coverpoints or covergroups yet, but it is primed for use in post-V2 test development.
-
-### PRE_VERIFIED_SUB_MODULES_V1
-
-Sub-modules that are pre-verified with their own testbenches have already reached V1 or higher stage.
+Structural coverage collection model must be checked in.
+This is a simulator tool specific file that captures what hierarchies and what types of coverage is collected.
+For example, pre-verified sub-mudules (including some `prim` components pre-verified thoroughly with FPV) can be black-boxed and only IO toggle coverage can be setup for those sub-modules for coverage collection.
+Functional coverage shell object must be created - this may not contain coverpoints or covergroups yet, but it is primed for use in post-V2 test development.
 
 ### TB_LINT_SETUP
 
 [VeribleLint](https://google.github.io/verible/verilog_lint.html) for the testbench is [setup]({{< relref "hw/lint/doc/README.md" >}}) to run in nightly regression, with appropriate waivers.
 * For DV testbench, an entry is expected to be added at `hw/<top-level-design>/lint/<top-level-design>_dv_lint_cfgs.hjson`
 * For FPV testbench, an entry is expected to be added at `hw/<top-level-design>/lint/<top-level-design>_fpv_lint_cfgs.hjson`
 
+### PRE_VERIFIED_SUB_MODULES_V1
+
+Sub-modules that are pre-verified with their own testbenches must have already reached V1 or higher stage.
+
 ### DESIGN_SPEC_REVIEWED
 
 RTL (uArch) specification reviewed and signed off.
+If a PRD document exists, the design specification must meet the product requirements.
```

Comment:
Can you expand PRD here? (I assume it's product design document or something, but I don't know and neither will other readers)

Reply:
@rswarbrick Product Requirement Document is my assumption - but since it confused at least one engineer it makes sense to expand